src/astro/sql/__init__.py

```diff
@@ -89,7 +89,7 @@ def append(
         columns=columns,
         casted_columns=casted_columns,
         **kwargs,
-    )
+    ).output
```

Comment:
@dimberman why we didn't have `output` before? Did it work?

Reply:
@tatiana without that `.output` there is no way to pass the result of the append to a validation function. Before the class was simply returning nothing. I'll make a ticket for this feature add, but this won't break any existing workflows.